### DISCUSSION TITLE
[求助] 如何在 博士论文模板里的学术成果清单部分 进一步区分SCI论文、EI论文、已授权专利、实审中专利等

### DISCUSSION THREAD

**Discussion by YoungZnBIT** [BITNP/BIThesis#451] [Q&A] [answered]
### 发生了什么？

在目前模板的pub.tex中，由于SCI和EI论文的类型都是article，所以不管怎么分都会划到同一个列表里，能否提供一个更为详细地模板举例说明一下怎么进一步划分自己的publication呢？

```tex
\printbibliography[heading=none,type=article,category=mypub,resetnumbers=true]
```

**Comment by YDX-2147483647** [comment, marked as answer]
可以通过`keyword`/`keywords`区分。

`misc/4_pub.tex`:

```latex
\printbibliography[…, keyword=dummy, …]{}
\printbibliography[…, keyword=sci, …]{}
```

https://github.com/BITNP/BIThesis/blob/743c8a7340f8697d75f69c843b80ef397e80fc44/templates/graduate-thesis/misc/4_pub.tex#L46

`reference/pub.bib`:

```bibtex
@article{
  …
  keywords = {dummy},
  …
}
@article{
  …
  keywords = {sci},
  …
}
```

https://github.com/BITNP/BIThesis/blob/743c8a7340f8697d75f69c843b80ef397e80fc44/templates/graduate-thesis/reference/pub.bib#L88

此外`printbibliography`还支持`notkeyword=<keyword>`、`subtype=<subtype>`等筛选选项，详情请参考 [biblatex 手册](http://mirrors.ctan.org/macros/latex/contrib/biblatex/doc/biblatex.pdf)。

**Comment by YoungZnBIT** [comment]
sorry，文档没有看仔细，solved，谢谢

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq/)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/bib-keyword.html
https://github.com/BITNP/BIThesis-wiki/blob/6a76c26ca78435550e068339ba1c31ce55622c69/wiki/faq/bib-keyword.md
